pkg/util/common.go

```diff
@@ -511,28 +511,31 @@ func ValidatePodsAreRunningInNamespace(clientset *kubernetes.Clientset, namespac
 		return fmt.Errorf("unable to list the pods in namespace %s due to %+v", namespace, err)
 	}
 
-	allPodExist := ValidatePodsAreRunning(clientset, pods)
-	if !allPodExist {
-		ValidatePodsAreRunningInNamespace(clientset, namespace)
+	timeout := time.NewTimer(10 * time.Minute)
```

Comment:
The total timeout should be configurable to handle greatest variance in cluster configuration. If that's a big ask the way things are now I'm fine committing this a logging a new issue to track for a later release.

Reply:
Currently for checking the postgres container status, we are having "postgresRestartInMins" parameter in synopsys-operator configmap, probably instead of adding into CRD, we can have it at synopsys-operator level @jdartigalongue